src/action-creators/logout.ts

```diff
@@ -21,6 +23,8 @@ const logout = (): Thunk => dispatch => {
   // clear state variables
   dispatch(clear())
 
+  dispatch(setRemoteSearch(false))
```

Comment:
Let's move `setRemoteSearch(false)` to the `clear` reducer so that we hide the details of resetting the state from `logout`.

Reply:
Sure.